kolibri/plugins/setup_wizard/api.py

```diff
@@ -32,6 +34,7 @@ def has_permission(self, request, view):
         return get_device_setting("subset_of_users_device")
 
 
+@method_decorator(csrf_protect, name="dispatch")
```

Comment:
I think maybe the facilityimport viewset below might also need this?

Reply:
yes, it does, i will add